commit 2a07954b83a3d4dc93031d3ce030fb9380a8e15a
Author: Kalle Valo <kalle.valo@iki.fi>
Date:   Fri Feb 27 07:50:00 2009 +0200

    at76c50x-usb: use dev_name() instead of struct device.bus_id
    
    Stephen Rothwell reported that bus_id from struct device will be removed, use
    dev_name() instead.
    
    Signed-off-by: Kalle Valo <kalle.valo@iki.fi>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

diff --git a/drivers/net/wireless/at76c50x-usb.c b/drivers/net/wireless/at76c50x-usb.c
index cc2a267..0c02f1c 100644
--- a/drivers/net/wireless/at76c50x-usb.c
+++ b/drivers/net/wireless/at76c50x-usb.c
@@ -2268,7 +2268,7 @@ static int at76_init_new_device(struct at76_priv *priv,
 
 	printk(KERN_INFO "%s: USB %s, MAC %s, firmware %d.%d.%d-%d\n",
 	       wiphy_name(priv->hw->wiphy),
-	       interface->dev.bus_id, mac2str(priv->mac_addr),
+	       dev_name(&interface->dev), mac2str(priv->mac_addr),
 	       priv->fw_version.major, priv->fw_version.minor,
 	       priv->fw_version.patch, priv->fw_version.build);
 	printk(KERN_INFO "%s: regulatory domain 0x%02x: %s\n",
